tools/site_admin/sitenews.php

```diff
@@ -330,8 +383,9 @@ function show_all_news_items_for_page( $news_page_id )
             }
             echo "</td>";
             echo "<td class='items default-border-bottom'>";
+            echo "<div class='news-header news-{$news_item['item_type']}'>" . html_safe($news_item['header']) . "</div>";
```

Comment:
intentionally no HTML header, right?

Reply:
Correct - I want to discourage SAs from putting HTML into the header. (I'd love for them to not put it in the content, but that ship has sailed.)